lib/grovepi/dht.ex

```diff
@@ -1,54 +1,45 @@
 defmodule GrovePi.DHT do
   alias GrovePi.Board
 
-  @moduledoc """
-  Read temperature and humidity from the Grove DHT sensor.
-
-  Example use:
+  use GrovePi.Poller, default_trigger: GrovePi.DHT.DefaultTrigger,
+  read_type: Digital.level
 
+  @moduledoc """
+  Listen for events from a GrovePi DHT (Digital Humidity and Temparature)
+  sensor. This module is configured for the DHT, the blue one, that comes
```

Comment:
I think this line should continue to reference the DHT11:

`This module is currently only configured with module_type 0 for the DHT11, ...`